fix(css): add bracketed range notation (#34670)
mdn/content: files/en-us/web/css/value_definition_syntax/index.md

- `smaller bold`, as `bold` is juxtaposed and must appear before the `smaller` keyword.
- `bold smaller bold`, as `bold` and `smaller` may only appear once.

> ## Bracketed range notation (`[,]`)

Reviewer: ```suggestion
## Bracketed range notation (`[min,max]`)
```

> Some properties take numeric values that can be in a limited range, and any value out of the range makes the declarations invalid and the declaration is ignored. For example, the [`column-count`](/en-US/docs/Web/CSS/column-count) property can have values from 1 to positive infinity only.

Reviewer: ```suggestion
Some types can accept numeric values within a certain range. For example, the [`column-count`](/en-US/docs/Web/CSS/column-count) property can accept an integer value between positive 1 and infinity, inclusive. The corresponding syntax looks like this:

<integer [1,∞]>

Any value outside this specified range causes the whole declaration to be invalid, therefore the browser will ignore it.
```

You should put "<integer [1,∞]>" inside a code fence. I didn't do it above because it would have caused the suggestion to break. That also uses a code fence ;-)

Author: ````suggestion
Some types can accept numeric values within a certain range. For example, the [`column-count`](/en-US/docs/Web/CSS/column-count) property can accept an integer value between positive 1 and infinity, inclusive. The corresponding syntax looks like this:

```plain
<integer [1,∞]>
```

Any value outside this specified range causes the whole declaration to be invalid, therefore the browser will ignore it.
````

Author: > You should put "<integer [1,∞]>" inside a code fence. I didn't do it above because it would have caused the suggestion to break. That also uses a code fence ;-)

**Pro tip:** Backticks in the code fence should be more than backticks in the code, e.g. to add the syntax section (3 backticks) here use four backticks for the suggestion fence ````` ````suggestion ````` (to show this example I used five backticks `````` ````` ````suggestion ````` ``````).

Reviewer: @OnkarRuikar ah, brilliant, I didn't know that — thanks for the tip!

> The _bracketed range notation_ `[min, max]` indicates the range between `min` and `max` values. The range includes `min` and `max` values as well. The range notation is used in numeric type notations, e.g. `<angle [0,180deg]>`. Infinities, -∞ and ∞, must not have units specified. Zero values can be written without units, even zero seconds.

Reviewer: ```suggestion
The _bracketed range notation_ `[min, max]` indicates an inclusive range between a `min` and `max` value. This notation is used in numeric type notations and can include units, e.g. `<angle [0,180deg]>`. Positive and negative Infinity (-∞ and ∞) must not have units specified. Types specified in units can have zero values specified with or without units, for example `<time [0s,10s]>` or `<time [0,10s]>`.
```

> Following are some examples:

Reviewer: ```suggestion
Here are some more examples:
```

> - `<integer>` or `<integer [-∞,∞]>` - any integer from negative infinity to positive infinity

Reviewer: ```suggestion
-  `<integer [-∞,∞]>`: Any integer from negative infinity to positive infinity is valid.
```

I wouldn't include `<integer>` — it doesn't include the range bracket notation. Or if you do want to include it, maybe include a line at the end of the bullet saying something like "Note that `<integer>` and `<integer [-∞,∞]>` are equivalent."

> - `<integer [0,∞]>` - negative integers not allowed

Reviewer: ```suggestion
- `<integer [0,∞]>`: Any integer from 0 to positive infinity is valid. Negative integers are invalid.
```

> - `<time [0s,10s]>` or `<time [0,10s]>` - time less than 11 seconds

Reviewer: ```suggestion
- `<time [0s,10s]>` or `<time [0,10s]>`: Any duration from 0 to 10 seconds is valid.
```

> - `<integer [-∞,-1]> | <integer [1,∞]>` - any integer except zero

Reviewer: ```suggestion
- `<integer [-∞,-1]> | <integer [1,∞]>`: Any integer except zero is valid.
```

      <td>Group must produce at least 1 value</td>
      <td><code>[ bold? smaller? ]!</code></td>
    </tr>
    <tr>
      <th colspan="4">Ranges</th>
    </tr>
    <tr>

>       <td><code>[,]</code></td>

Reviewer: ```suggestion
      <td><code>[min,max]</code></td>
```

Maybe update the curly braces to be consistent with this too? I think min and max are pretty intuitive; more so than A and B.
